libs/core/langchain_core/tracers/langchain.py

```diff
@@ -112,6 +109,11 @@ def __init__(
         self.tags = tags or []
         self.latest_run: Optional[Run] = None
 
+    def _start_trace(self, run: Run) -> None:
+        super()._start_trace(run)
+        if run._client is None:
+            run._client = self.client
```

Comment:
This is so that if you were defining the client solely via a LangChainTracer we will hand off that client to the nested run tree

Reply:
any situations when one might create two tracers with different clients and pass them as callbacks?

Reply:
I assume no easy way to catch that as an issue to raise an error?

Reply:
hmm I kinda would prefer if we still traced something / didn't raise an error that could potentially interrupt their execution flow? It's kinda hard to catch correctly